frontend/src/components/NotificationPanel.jsx

```
@@ -3,12 +1,165 @@
+      day: "numeric",
+      hour: "2-digit",
+      minute: "2-digit",
+    });
+  };
+
+  if (loading) {
+    return (
+      <div className={styles.container}>
+        <h3>NOTIFICATIONS</h3>
+        <div className={styles.loadingDots}>
+          <div className={styles.dot}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)